schematron/v2.2.0/v2-2-0_L200_Audit.sch

```diff
@@ -1016,6 +1017,13 @@
       <sch:assert test="//auc:TimeSeriesData/auc:TimeSeries[@ID = current()/@IDref and auc:ResourceUseID/@IDref = current()/ancestor::auc:ResourceUse/@ID and auc:ReadingType/text() = 'Total']" role="">Each auc:LinkedTimeSeriesID must point to an auc:TimeSeries that (1) points to the same auc:ResourceUse through auc:ResourceUseID and (2) has an auc:ReadingType of Total</sch:assert>
     </sch:rule>
   </sch:pattern>
+  <sch:pattern see="6.2.2.1 (d)" id="submetering">
+    <sch:title>Submetering</sch:title>
+    <sch:rule context="/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios/auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured]/auc:ResourceUses[auc:MeterID]">
+      <sch:assert test="auc:ParentResourceUse" role="">auc:ParentResourceUse</sch:assert>
```

Comment:
Hey ted so we want to check:
1. That all `auc:ResourceUse[auc:EndUse='All end uses']` declare an `auc:MeterID` (i.e. an end use meter at the building we are intending to be linked to some sort of utility and hence have a meterID
2. That all `auc:ResourceUse[not(auc:EndUse='All end uses')]` declare an `auc:ParentResourceUse`

I don't think this check captures this - does it?

Reply:
Ahh ok I understand now. So ResourceUses that aren't All end uses should _not_ specify auc:MeterID?

Reply:
Created issue here to better understand our submetering model #46